### PR TITLE
spdlog 0.11.0 (new formula)

### PR DESCRIPTION
**Pull request by ahundt**
Add spdlog 0.11.0 in spdlog.rb  which is "a super fast C++ logging library". Found at https://github.com/gabime/spdlog

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by ahundt** [comment]
I'm not really sure what to do about the error here, is atomic really not implemented for all three test versions of OS X? Is there a specific flag, setting or something else that should be set for C++11 only libraries?
```
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/atomic:543:2: error: <atomic> is not implemented
#error <atomic> is not implemented
 ^
```

File: Formula/spdlog.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
+class Spdlog < Formula
+  desc "Super fast C++ logging library."
+  homepage "https://github.com/gabime/spdlog"
+  url "https://github.com/gabime/spdlog/archive/v0.11.0.tar.gz"
+  sha256 "8c0f1810fb6b7d23fef70c2ea8b6fa6768ac8d18d6e0de39be1f48865e22916e"
+  head "https://github.com/gabime/spdlog.git", :branch => "master"
```
Comment (woodruffw) [inline Formula/spdlog.rb]: We assume master branch by default, removing it here should be fine :smile:

Comment (ahundt) [inline Formula/spdlog.rb]: can that just be a later patch if desired?

Comment (woodruffw) [inline Formula/spdlog.rb]: You can just commit it to this PR, if that's what you mean.

Comment (ahundt) [inline Formula/spdlog.rb]: done

Comment (woodruffw) [inline Formula/spdlog.rb]: Much appreciated. That's my only nit on this formula, the rest LGTM to me. We'll leave it open for a bit to give other maintainers an opportunity to provide feedback.

**Comment by woodruffw** [comment]
Looks good to me. Could you squash this down to a single commit? Thanks!

File: Formula/spdlog.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  sha256 "8c0f1810fb6b7d23fef70c2ea8b6fa6768ac8d18d6e0de39be1f48865e22916e"
+  head "https://github.com/gabime/spdlog.git"
+
+  depends_on "cmake" => :build
+
+  needs :cxx11
+
+  def install
+    ENV.cxx11
+    ENV.universal_binary if build.universal?
```
Comment (MikeMcQuaid) [inline Formula/spdlog.rb]: Please remove this, thanks.

**Comment by ahundt** [comment]
should be good now!

**Comment by woodruffw** [comment]
Thanks! This looks good to me, but I'll leave it open for a bit in case other maintainers have things to add.

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by woodruffw** [comment]
Thanks @ahundt, and congratulations on your first formula! :tada:

**Pull request closed without merging**